mmcv/runner/hooks/momentum_updater.py

```diff
@@ -404,11 +426,13 @@ class OneCycleMomentumUpdaterHook(MomentumUpdaterHook):
     """
 
     def __init__(self,
-                 base_momentum=0.85,
-                 max_momentum=0.95,
-                 pct_start=0.3,
-                 anneal_strategy='cos',
-                 three_phase=False,
+                 base_momentum: Union[float, List[float], Dict[str,
+                                                               float]] = 0.85,
```

Comment:
The indent is weird, will this format better?
```suggestion
                 base_momentum: Union[
                     float, List[float], Dict[str, float]] = 0.85,
```